modules/core/src/main/java/org/apache/ignite/IgniteSnapshot.java

```diff
@@ -69,9 +70,10 @@ public interface IgniteSnapshot {
      * In-memory cache groups also supported.
      *
      * @param name Dump name.
+     * @param cacheGroupNames Cache groups to include in snapshot or {@code null} to include all.
      * @return Future which will be completed when dump ends.
      */
-    public IgniteFuture<Void> createDump(String name);
+    public IgniteFuture<Void> createDump(String name, @Nullable Set<String> cacheGroupNames);
```

Comment:
Replace Set with Collection, as this style used in `restoreSnapshot` methods.

Reply:
Fixed.